ansible-tower-rhel/azuredeploy.json

```diff
@@ -0,0 +1,728 @@
+         "apiVersion":"2015-06-15",
+         "location":"[resourceGroup().location]",
+         "comments":"This Public IP is used by the Client VM 02",
+         "tags":{
+            "displayName":"Public IP Address -Client VM-02"
+         },
+         "properties":{
+            "publicIPAllocationMethod":"Static",
+            "dnsSettings":{
+               "domainNameLabel":"[variables('pipClient02DnsName')]"
+            }
+         }
+      },
+      {
+         "type":"Microsoft.Network/networkInterfaces",
+         "name":"[variables('vmClient01NicName')]",
+         "apiVersion":"2015-06-15",
+         "location":"[resourceGroup().location]",
+         "comments":"These NIC will be used by Client VM 01",
+         "tags":{
+            "displayName":"NIC Client VM-01"
+         },
+         "dependsOn":[
+            "[resourceId('Microsoft.Network/virtualNetworks', variables('ansibleVnetName'))]",
+            "[resourceId('Microsoft.Network/publicIPAddresses', variables('pipClient01Name'))]"
+         ],
+         "properties":{
+            "ipConfigurations":[
+               {
+                  "name":"ipconfig1",
+                  "properties":{
+                     "privateIPAllocationMethod":"Static",
+                     "privateIPAddress":"10.0.1.21",
+                     "subnet":{
+                        "id":"[variables('clientSubnetRef')]"
+                     },
+                     "publicIPAddress":{
+                        "id":"[resourceId('Microsoft.Network/publicIPAddresses', variables('pipClient01Name'))]"
+                     }
+                  }
+               }
+            ]
+         }
+      },
+      {
+         "type":"Microsoft.Network/networkInterfaces",
+         "name":"[variables('vmClient02NicName')]",
+         "apiVersion":"2015-06-15",
+         "location":"[resourceGroup().location]",
+         "comments":"These NIC will be used by Client VM 02",
+         "tags":{
+            "displayName":"NIC Client VM-02"
+         },
+         "dependsOn":[
+            "[resourceId('Microsoft.Network/virtualNetworks', variables('ansibleVnetName'))]",
+            "[resourceId('Microsoft.Network/publicIPAddresses', variables('pipClient02Name'))]"
+         ],
+         "properties":{
+            "ipConfigurations":[
+               {
+                  "name":"ipconfig1",
+                  "properties":{
+                     "privateIPAllocationMethod":"Static",
+                     "privateIPAddress":"10.0.1.22",
+                     "subnet":{
+                        "id":"[variables('clientSubnetRef')]"
+                     },
+                     "publicIPAddress":{
+                        "id":"[resourceId('Microsoft.Network/publicIPAddresses', variables('pipClient02Name'))]"
+                     }
+                  }
+               }
+            ]
+         }
+      },
+      {
+         "type":"Microsoft.Compute/virtualMachines",
```

Comment:
do client VMs need in-guest config?

Also, consider a copy loop for the 2 client VMs - removes a lot of code...